server/handler.go

```diff
@@ -35,8 +36,15 @@ func (h *Handler) ConnectionClosed(c *mysql.Conn) {
 }
 
 // ComQuery executes a SQL query on the SQLe engine.
-func (h *Handler) ComQuery(c *mysql.Conn, query string, callback func(*sqltypes.Result) error) error {
-	schema, rows, err := h.e.Query(query)
+func (h *Handler) ComQuery(
+	c *mysql.Conn,
+	query string,
+	callback func(*sqltypes.Result) error,
+) error {
+	// TODO: create proper session
+	session := sql.NewBaseSession(context.TODO())
```

Comment:
Should this session be provided as a ComQuery param?

Reply:
The handler conforms mysql.Handler interface. Changing the signature of ComQuery breaks it.

https://github.com/src-d/go-vitess/blob/master/mysql/server.go#L76